File: 1-two-sum/two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        num_len = len(nums)
        for i in range(num_len):
            for j in range(i+1, num_len):
                if nums[i]+nums[j] == target:
                    return [i,j]
        else:
            return 0

        

File: 1-two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer, Integer> mapset = new HashMap<>();
        for(int i =0;i<nums.length;i++){
            int comp = target - nums[i];

            if(mapset.containsKey(comp)){
                return new int[] {mapset.get(comp), i};
            }
            mapset.put(nums[i],i);
        }
        return new int[] {};
    }
}